helix-view/src/editor.rs

```diff
@@ -213,6 +214,7 @@ impl Default for StatusLineConfig {
             left: vec![E::Mode, E::Spinner, E::FileName],
             center: vec![],
             right: vec![E::Diagnostics, E::Selections, E::Position, E::FileEncoding],
+            separator: String::new(),
```

Comment:
Should default to `|` or better yet `│` (the box drawing character)

Reply:
Can do.